archive/tar/tar_test.go

```diff
@@ -131,18 +134,93 @@ func unpackIt(a archive.Archive, src string) error {
 	return a.Unpack("/dev/null", reader)
 }
 
+func createGoodTarfile() {
```

Comment:
I'm a bit confused as to why this is here. Shouldn't it just write the files and then the tar code creates the tar.